README.md

```diff
@@ -22,10 +22,125 @@ The lack of a standardized, cryptographic verification process for exchange solv
 
 ## Specification
 
+The key words “MUST”, “MUST NOT”, “REQUIRED”, “SHALL”, “SHALL NOT”, “SHOULD”, “SHOULD NOT”, “RECOMMENDED”, “MAY”, and “OPTIONAL” in this document are to be interpreted as described in RFC 2119.
+
 ### Solidity Interface
 
 ```solidity
-// [Sample Interface showing methods for proof of solvency]
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity ^0.8.18;
+
+/**
+    @title Cryptographically Auditable Exchange standard
+    @dev See https://eips.ethereum.org/EIPS/eip-xxxx
+ */
+interface ICryptographicallyAuditable {
+    /**
+     * @dev Struct representing an address ownership proof submitted by the CEX
+     * @param cexAddress MUST be the address owned by the CEX (submitted as a string, as it can be a non-EVM address)
+     * @param chain MUST be the name of the chain name where the address belongs (e.g., ETH, BTC)
```

Comment:
Is there any standardized nomenclature to define a chain that we can point to?

Reply:
Found this standard: https://www.iso.org/news/ref2729.html , but can't find a registry with the actual values :)